xkcd/pkg/pbuff/convert.go

```diff
@@ -0,0 +1,60 @@
+package pbuff
+
+import "github.com/vespian/go-exercises/xkcd/pkg/types"
+
+func PBStoryFromStory(s *types.Story) *PBStory {
+	res := PBStory{
```

Comment:
Not if it was my repo - no - I find one-liner returns where the thing being initialized is immediately returned to be much simpler to read than one that introduces the "mental indirection" of a variable, for no reason. That's just me, which is why i say "not if it was my repo". But if it was, say, vault or secrets I would approve the PR, yes